tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -47,10 +47,11 @@ struct choose_literal<wchar_t> {
 
 template <class charT, class... Args>

```

Comment:
Why is this so pretty?

```suggestion
template <class _CharT, class... Args>
```

Reply:
This is `pretty` test code ✨

Reply:
Then it shouldb't `beUgly` then :P